legal/new-folder-ISDA/Credit Support Annex

```diff
@@ -0,0 +1,423 @@
+CREDIT SUPPORT ANNEX (to the Schedule to the 2002 ISDA Master Agreement)
```

Comment:
@votre-will this new folder needs to be renamed, lowercase only please. The files that you added, also need to be renamed to be lowercase and also have the file extension of ".md". They are all not currently showing up in the preview link.

https://docs-git-fork-votre-will-will-docs-update-votre.vercel.app

I would do this for you, but since you forked it, I am not able to edit this branch.